<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkCESTStatisticsViewControls</class>
 <widget class="QWidget" name="QmitkCESTStatisticsViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>963</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please select an image!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="threeDimToFourDimPushButton">
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Copy first time step to rest</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="normalizeImagePushButton">
     <property name="text">
      <string>Normalize CEST image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_StatisticsGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Statistics</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QTableWidget" name="m_StatisticsTable">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="autoScroll">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::DotLine</enum>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="rowCount">
         <number>7</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>25</number>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>Mean</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>StdDev</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>RMS</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Max</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Min</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>N</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>V (mmÂ³)</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>0</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_CopyStatisticsToClipboardPushButton">
           <property name="text">
            <string>Copy to Clipboard</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="fixedRangeCheckBox">
       <property name="text">
        <string>Define manual range</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="fixedRangeLowerLabel">
       <property name="text">
        <string>Lower limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="fixedRangeLowerDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <double>-299.899999999999977</double>
       </property>
       <property name="maximum">
        <double>299.990000000000009</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fixedRangeUpperLabel">
       <property name="text">
        <string>Upper limit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="fixedRangeUpperDoubleSpinBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>-299.899999999999977</double>
       </property>
       <property name="maximum">
        <double>299.990000000000009</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_PlotGroupBox">
     <property name="title">
      <string>Plot</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QmitkPlotWidget" name="m_DataViewWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>220</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QmitkPlotWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPlotWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
